### PR TITLE
Hidden columns

### PR DESCRIPTION
**Pull request by Lunalo**

### CODE REVIEW

File: instat/static/InstatObject/R/data_object_R6.R [hunk 1]
```diff
@@ -1153,7 +1153,7 @@ data_object$set("public", "get_data_type", function(col_name = "") {
 }
 )
 
-data_object$set("public", "set_hidden_columns", function(col_names) {
+data_object$set("public", "set_hidden_columns", function(col_names=c()) {
```
Comment (dannyparsons) [inline instat/static/InstatObject/R/data_object_R6.R]: spacing needs fixing around the =

File: instat/static/InstatObject/R/instat_object_R6.R [hunk 2]
```diff
@@ -844,7 +844,7 @@ instat_object$set("public","copy_data_frame", function(data_name, new_name) {
 } 
 )
 
-instat_object$set("public","set_hidden_columns", function(data_name, col_names) {
+instat_object$set("public","set_hidden_columns", function(data_name, col_names=c()) {
```
Comment (dannyparsons) [inline instat/static/InstatObject/R/instat_object_R6.R]: and same here

**Comment by Lunalo** [comment]
@dannyparsons  You are absolutely right on this. I will still implement the condition in the receiver.

**Comment by dannyparsons** [comment]
You can do that on another branch or this one if you like.

**Comment by Lunalo** [comment]
Yes ,  I can do this on the other branch

**Comment by maxwellfundi** [comment]
@dannyparsons i thinkthis is ready for you. Requested changes on spacing has been done

**Pull request merged**